src/ontology/foodon-edit.owl

```diff
@@ -56067,6 +56229,8 @@ SubClassOf(obo:FOODON_03420101 obo:FOODON_03420144)
 # Class: obo:FOODON_03420102 (germinated or sprouted seed)
```

Comment:
Could you ask PO plant ontology if they have a term for this?  They likely do.

Reply:
I think they do not have one, but I'll check with them. Until that, I think we can update this existing term.